app/views/flights/index.html.erb

```diff
@@ -0,0 +1,18 @@
+<h1>All Flights</h1>
+<section id="flights">
+  <% @flights.each do |flight| %>
+    <section id="flight-<%= flight.id %>">
+      <h2><%= "Flight Number: #{flight.number}" %></h2>
+      <p><%= "Flight Airline: #{flight.airline_name}" %></p>
+      <h3>Passengers on this Flight</h3>
+      <section id="flight_passengers">
+        <% flight.passengers.each do |passenger| %>
+          <section id="passenger-<%= passenger.id %>">
+            <p><%= passenger.name %></p>
+            <%= button_to "Remove From Flight", flight_passenger_path(flight.id, passenger.id), method: :delete %>
+          </section>
+        <% end %>  
+      </section>
+    </section>
+  <% end %>
+</section>
```

Comment:
It's ok to use white space in your html files to section out different logic, so the eye doesn't need to look at a wall of text. ;) Nice job!